portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,28 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Abakir's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
+    <!--nav-->
+    <div id="nav">
+        <a href="#home">Home</a>
+        <a href="#aboutMe">About Me</a>
+        <a href="./projects.html">Projetcs</a>
+        <a href="#contact">Contact</a>
```

Comment:
I don't think this breaks anything, but these hrefs don't correspond to any id in file.

Reply:
Yes I did this because, I'm planning to link them to different files, but haven't done so yet in this pull request

Reply:
Ack.